### PR TITLE
[WIP] Add Aishell-2 ASR Recipe for Espnet2

### PR DESCRIPTION
**Pull request by jctian98**
Hi, dear Espnet team.
I'm trying to make a PR for Espent2 Aishell-2 recipe.
link for the dataset: https://www.aishelltech.com/aishell_2

### CODE REVIEW

**Comment by sw005320** [comment]
LGTM.
@ftshijt, can you take a look at this?

Once it is ready, we may merge this PR and have a follow-up PR to update a result and upload a model.

File: egs2/aishell2/asr1/local/data.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+. ./path.sh
+. ./cmd.sh
+. ./db.sh
+
+
+if [ -z "${AISHELL2}" ]; then
+  log "Error: \$AISHELL2 is not set in db.sh."
+  exit 2
+fi
+
```
Comment (ftshijt) [inline egs2/aishell2/asr1/local/data.sh:34]: Could you mention somewhere to show where the AISHELL2 data can be requested? That could be very helpful for those interested in doing the recipe.

File: egs2/aishell2/asr1/local/data.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+. utils/parse_options.sh
+
+if [ $# -ne 0 ]; then
+    log "Error: No positional arguments are required."
+    exit 2
+fi
+
+. ./path.sh
+. ./cmd.sh
+. ./db.sh
```
Comment (ftshijt) [inline egs2/aishell2/asr1/local/data.sh:27]: Recommend to put these lines to before utils/parse_options.sh to allow user to change it from shell argument

**Review by ftshijt — COMMENTED** [review]
Thanks for your contribution! I have some minor issues and suggestions as follows:

**Comment by jctian98** [comment]
> 

All fixed. Thanks :)

**Comment by ftshijt** [comment]
Everything looks find for me. After the CI checks, I will merge it for now as we already got some initial results. You can start another follow-up PRs for other models and results. Many thanks!

**Comment by sw005320** [comment]
Thanks, @jctian98!

**Pull request merged**